src/commands/orders.py

```diff
@@ -0,0 +1,197 @@
+    sellTokenDecimals, sellTokenLabel = sellToken['decimals'], format_token_short(sellToken)
+    buyTokenDecimals, buyTokenLabel = buyToken['decimals'], format_token_short(buyToken)
+
+    labelColor = COLOR_LABEL
+    if cancelDate is None:
+      cancelDateText = ''
+    else:
+      labelColor = COLOR_LABEL_DELETED
+      cancelDateText = click.style('  Cancel date', fg=labelColor) + ': ' + click.style(format_date_time(cancelDate), bg=COLOR_LABEL_DELETED) + '\n'
+
+    if deleteDate is None:
+      deleteDateText = ''
+    else:
+      labelColor = COLOR_LABEL_DELETED
+      deleteDateText = click.style('  Deleted date', fg=labelColor) + ': ' + click.style(format_date_time(deleteDate), bg=COLOR_LABEL_DELETED) + '\n'
+
+    tradePriceText = ''
+    if soldVolume > 0:
+      tradePriceText = (
+        click.style(f'  Avg. Traded Price {sellTokenLabel}/{buyTokenLabel}', fg=labelColor) + ': ' + 
+        format_price(
+          calculate_price(
+            numerator=boughtVolume,
+            denominator=soldVolume,
+            decimals_numerator=buyTokenDecimals,
+            decimals_denominator=sellTokenDecimals
+          ), 
+          currency=buyTokenLabel
+        ) +
+        '\n' +
+        click.style(f'  Avg. Traded Price {buyTokenLabel}/{sellTokenLabel}', fg=labelColor) + ': ' + 
+        format_price(
+          calculate_price(
+            numerator=soldVolume,
+            denominator=boughtVolume,
+            decimals_numerator=sellTokenDecimals,
+            decimals_denominator=buyTokenDecimals
+          ),
+          currency=sellTokenLabel
+        ) +
+        '\n'
+      )
+
+    click.echo(
+      click.style('  Order date', fg=labelColor) + ': ' + 
+      format_date_time(order['createDate']) + '\n' +       
+      cancelDateText + 
+      deleteDateText + 
+      '\n' + 
+
+      click.style('  Trader', fg=labelColor) + ': ' + 
+      order['owner_address'] + '\n' + 
+
+      click.style('  Order Id', fg=labelColor) + ': ' + 
+      format_integer(order['order_id']) + '\n' + 
+
+      click.style('  From batch', fg=labelColor) + ': ' + 
+      format_batch_id_with_date(order['fromBatchId']) + '\n' +
+
+      click.style('  To batch', fg=labelColor) + ': ' +
+      format_batch_id_with_date(order['untilBatchId']) + 
+      '\n\n' + 
+
+      click.style('  Sell Token', fg=labelColor) + ': ' + 
+      format_token_long(sellToken) + '\n' + 
+
+      click.style('  Buy Token', fg=labelColor) + ': ' + 
+      format_token_long(buyToken) + '\n' +
+
+      click.style('  Sold volume', fg=labelColor) + ': ' + 
+      format_amount_in_weis(soldVolume, sellTokenDecimals) + ' ' + sellTokenLabel + '\n' + 
+
+      (
+        click.style('  Bought volume', fg=labelColor) + ': ' + 
+        format_amount_in_weis(boughtVolume, buyTokenDecimals) + ' ' + buyTokenLabel + '\n'
+        if soldVolume else ''
+      ) + # TODO: Add percentage https://github.com/gnosis/dex-cli/issues/31
+      '\n' +
```

Comment:
Would it be too much to also add the order size? Meaning, max buy/sell amounts, not just the matched amounts.

Reply:
This was added in a different PR. I added sell amount. Take a look if you miss the max receiving amount. Probably we should add it as you say

Reply:
https://github.com/gnosis/dex-cli/issues/39